sroka/api/google_ad_manager/README.md

```diff
@@ -52,6 +56,6 @@ stop_date = {'year': year,
              'month': end_month,
              'day': end_day}
 
-data = get_data_from_admanager(query, dimensions, columns, start_date, stop_date, custom_field_id, network_code=1234)
+data = get_data_from_admanager(query, dimensions, columns, start_date, stop_date, custom_field_id, dimension_attributes, network_code=1234)
```

Comment:
Could you please add values to both `custom_field_id=custom_field_id` and `dimension_attributes=dimension_attributes` indicating they are key word arguments, as it's done for `network_code`?

Reply:
Done!